api/user_services.proto

```diff
@@ -91,6 +91,12 @@ service UserRegistry {
       delete: "/users/{user_id}/purge"
     };
   };
+  // Transfers solely owned entity rights to another user
+  rpc TransferOwnedEntityRights(TransferEntityRightsRequest) returns (google.protobuf.Empty) {
```

Comment:
What is solely owned? Rights that only the user identified by `user_ids` possesses?

What should be clear from the description (and rpc name) is what it does, i.e. does it:
1. Transfer only rights that `user_ids` has, and nobody else has?
2. Transfer all rights that `user_ids` has

Reply:
yeah I didn't come up with a concise name that explains the issue. `Transfering rights for all entities where the user is the only collaborator that has full rights on the entity` would explain it but not in a concise way.

Reply:
I actually like @johanstokking's (2). If we just make it transfer all rights, that will be a lot easier to explain.

Reply:
@htdvisser By transferring all the rights you mean the full (owner) rights? Because if you mean all the entity rights then the user might not have the rights to set the collaborators for the entity and this doesn't seem like something we should bypass.